content/docs/searching.md

```diff
@@ -316,7 +313,7 @@ The following commands, available through the Search menu or keyboard shortcuts,
             - uses _Wrap around_ setting from _Find_ window
             - uses _Search mode_ = _Normal_ (regardless of its current setting in the _Find_ window)
```

Comment:
Unless there was a recent change that I'm not aware of, this is incorrect. It's not the case in 8.1.9, 8.4.5 & 8.4.6. In fact (amusingly enough) only in the last few days did I discover the convenience of using this action when tuning or testing a regex: a single keyboard action launches a regex find, and delightfully, focus stays with the file tab. This is so nice I was thinking it deserves explicit mention, but probably not in this section (which should remain regex-innocent), but in the regex section.

Reply:
@NeilS-Wpg It WAS a recent change.  This was never meant to take anything but literal text from a selection to do the find.  See #3229

Reply:
Well, that's funny-sad, because when I realized I could do it (as I described it above) I felt like a kid at the beach who found money in the sand, and this glorious feature which has been around for years has just days ago been withdrawn... cosmically perhaps at the same instant that I discovered it.

It's not that I have a great dislike of working with the Find dialog, it's that I hate having to place focus back on the file after a find operation brings something of interest in view. (I believe I have a solution for this but it's likely not fully baked.)

Anyway, thanks (but NO THANKS!) for the info.

Reply:
Maybe I'm not seeing the full picture here but I don't see much in the way of substantial discussion of the issue.

At https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3229 we have one user complaint in 2017, a year later another user posts in agreement, and a year after that, Scott joins in. And in saying:

> So for good or bad, it seems like these features (interactive Find, Select-and-Find, (keyboard)Find-next) might be inexorably tied together...

he essentially makes the point that "no matter what, you're gonna get burned", IOW, there will be times when these searches don't do what you were wanting.

The gold-plated full meal deal would be independent options pertaining to each subtype of search, and it's easy to imagine the np++ brain trust concluding it's not important enough to justify all the extra clutter.

So what prompted this very low intensity issue (with two "outside" participants in seven years, and being almost four years dormant) to get promoted to the point of being deemed worthy of action?

Reply:
>...independent options pertaining to each subtype of search, and it's easy to imagine the np++ brain trust concluding it's not important enough to justify all the extra clutter.

Spot on!

>So what prompted this very low intensity issue ... to get promoted to the point of being deemed worthy of action?

This (very recent) user complaint: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/13145